docs/getting-started.md

```diff
@@ -148,7 +148,6 @@ While you can use any editor of your choice to develop your app, you will need t
 We recommend installing Node, Watchman, and JDK using [Homebrew](http://brew.sh/). Run the following commands in a Terminal after installing Homebrew:
 
 ```
-brew install yarn
```

Comment:
Yarn is used to start the packager and install dependencies.

What do you think about letting this brew install, but keeping your other changes?

Reply:
Ok that's fine. I've put it back.

I thought `npm` is used if `yarn` is not found when installing deps, but still it's needed to start the packager.